dev-docs/source/CLion.rst

```diff
@@ -119,23 +130,35 @@ Debugging with CLion
 
 To debug workbench, you'll need to edit the ``workbench`` CMake Application configuration.
 
-#. Set the executable to be the ``python.exe`` in your conda installation
+#. Set the executable to be the ``python`` executable in your conda installation:
 
    .. hlist::
       :columns: 1
 
-      - On Linux: ``/path/to/mambaforge/envs/mantid-developer/bin/python.exe``
+      - On Linux & macOS: ``/path/to/mambaforge/envs/mantid-developer/bin/python``
       - On Windows: ``/path/to/mambaforge/envs/mantid-developer/python.exe``
 
-#. Set the program arguments
+#. Set the program arguments:
 
    .. hlist::
       :columns: 1
 
-      - On Linux: ``workbench --single-process``
-      - On Windows: ``workbench-script.pyw --single-process``
+      - On Linux & macOS: ``workbench --single-process``
```

Comment:
```suggestion
      - On Linux & macOS & Windows: ``workbench --single-process``
```
Then the second line ("On Windows") could just get removed.

Reply:
Sorry for this last minute comment, on my CLion configuration on mac, mantid only launches  from the cmake configuration I set program arguments to : ``-m workbench --single-process`` . 
To make it work as in pycharm, is necessary to create a python run configuration.

Reply:
Thanks for that @adriazalvarez . I don't use CLion so am not best placed to make these changes. Would you be able to add a commit to implement that?

Reply:
@thomashampson I only use it on MacOS, I don't know if they same applies to other systems.